### ISSUE TITLE
Hive is being forced in 1.2.0

### ISSUE THREAD

**Issue by BillCartwright** [sparklyr/sparklyr#2460]
After upgrading to version 1.2.0 from 1.1.0 my previously working sparklyr connection is throwing Hive class errors.  It seems to force Hive as to looks like enableHiveSupport() is being called no matter what.  I tried setting spark.sql.catelogImplementation to "in-memory" in config but that doesn't seem to do anything/work.

Thanks

The error is "Unable to instantiate SparkSession with Hive support because Hive classes are not found.

![Screen Shot 2020-05-05 at 11 22 53 AM](https://user-images.githubusercontent.com/64862723/81114310-a4f6ba80-8ed6-11ea-9b8f-ac6b9fe65759.png)

![Screen Shot 2020-05-05 at 11 23 18 AM](https://user-images.githubusercontent.com/64862723/81114223-7842a300-8ed6-11ea-99f5-83ce925256fb.png)

**Comment by yl790** [comment]
@BillCartwright Hey actually there is not really any officially supported way in sparklyr to skip Hive integration yet, but I imagine it would not be difficult to implement one.

There was some inconsistency in sparklyr 1.1.0 where some code path instantiating `HiveContext` for Spark version 1.* was not doing the equivalent for Spark 2.* or above (i.e., calling `enableSparkSupport` on `SparkSession.Builder`). This was fixed in sparklyr 1.2.0 but apparently that broke your usual workflow. Sorry :/

I plan to have a pull request to address this issue (hopefully soon, and definitely should be addressed before sparklyr 1.3 release I think).

**Comment by yl790** [comment]
@BillCartwright You can now run the following to have a spark connection without enabling hive support

```
devtools::install_github("yl790/sparklyr", ref = "skip_hive_integration")

library(sparklyr)

config <- spark_config()
config$sparklyr.connect.enablehivesupport <- FALSE # default will be TRUE for backward compatibility reasons

sc <- spark_connect(master = "local", config = config)
```

**Comment by BillCartwright** [comment]
Thanks!  Will this fix be scheduled for 1.3 release?

**Comment by yl790** [comment]
@BillCartwright Yes definitely.
